Gruntfile.js

```diff
@@ -413,12 +426,22 @@ module.exports = function(grunt) {
 		grunt.task.run(['bump-only:' + versionReleaseType, 'dist', 'replace:readme']);
 	});
 
+
+	/* -------------
+		SERVEFAST
+	------------- */
+	grunt.registerTask('servefast', 'Serve the files with no "dist" build or tests. Optional --no-less to also disabled compiling less into css.', function() {
```

Comment:
Thanks, I couldn't figure out on a Friday afternoon what the heck was going on with travis errors.

Nor am I sure why or how they were introduced.

I am gonna dig in and look at what happened, but... you didn't remove the enhancements to "servefast" did you?

Those kinda sneaked in this, but were the result of an afternoon playing with it with @interactivellama .